ddsc/core/ddsapi.py

```diff
@@ -8,6 +9,8 @@
 AUTH_TOKEN_CLOCK_SKEW_MAX = 5 * 60  # 5 minutes
 SETUP_GUIDE_URL = "https://github.com/Duke-GCB/DukeDSClient/blob/master/docs/GettingAgentAndUserKeys.md"
 RESOURCE_NOT_CONSISTENT_RETRY_SECONDS = 2
+SERVICE_DOWN_RETRY_SECONDS = 60  # 1 minute
+SERVICE_DOWN_MESSAGE = "DukeDS service is down. Will continue retrying."
```

Comment:
Can we elaborate on the message to give a little more guidance? It would also be useful to print the time the message occurred.

```
Duke Data Service is currently unavailable, so this operation cannot complete right now. The operation will be retried automatically, so no action is required. It will complete once Duke Data Service is available.

To cancel this operation, press Ctrl+C.
```